mmv1/templates/terraform/resource.html.markdown.erb

```diff
@@ -173,7 +172,7 @@ In addition to the arguments listed above, the following computed attributes are
 ## Timeouts
 
 This resource provides the following
-[Timeouts](/docs/configuration/resources.html#timeouts) configuration options:
+[Timeouts](https://developer.hashicorp.com/terraform/plugin/sdkv2/resources/retries-and-customizable-timeouts) configuration options:
```

Comment:
Can't use the relative link anymore as docs have moved away from the terraform.io domain